### PR TITLE
Let user return envs slice

### PR DESCRIPTION
**Pull request by potsbo**
## WHY

in https://github.com/wantedly/developers-account-mapper/pull/13 env vars are rendered in exec command.
However, this should be handled by user model it self.

## WHAT

- [x] Implement Envs()
- [x] Add test for it

### CODE REVIEW

File: models/user_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+	if len(actual) != len(expect) {
+		t.Fatalf("%v does not much to expected: %v", len(actual), len(expect))
+	}
+
+	for i := range expect {
+		if expect[i] != actual[i] {
+			t.Fatalf("%v does not much to expected: %v", actual, expect)
+		}
+	}
```
Comment (dtan4) [inline models/user_test.go:39]: To compare two slices, you can use [`reflect.DeepEqual`](https://golang.org/pkg/reflect/#DeepEqual). This checks whether two slices have the same elements or not.

File: store/dynamo_db.go [hunk 2]
```diff
@@ -83,17 +85,17 @@ func (d *DynamoDB) GetUserByLoginName(loginName string) (*models.User, error) {
...
-		gitHubUsername = *resp.Item["GitHubUsername"].S
+		user = &models.User {
+			LoginName:      loginName,
+			GitHubUsername: *resp.Item["GitHubUsername"].S,
+			SlackUserId: *resp.Item["SlackUserId"].S,
+			SlackUsername: *resp.Item["SlackUsername"].S,
+		}
 	} else {
 		return nil, fmt.Errorf("%s is not registered yet", loginName)
 	}
```
Comment (dtan4) [inline store/dynamo_db.go:90]: refactoring:

```go
	if len(resp.Item) != 1 {
		return nil, fmt.Errorf("%s is not registered yet", loginName)
	}

	user := &models.User {
		LoginName:      loginName,
		GitHubUsername: *resp.Item["GitHubUsername"].S,
		SlackUserId: *resp.Item["SlackUserId"].S,
		SlackUsername: *resp.Item["SlackUsername"].S,
	}
```

Comment (dtan4) [reply store/dynamo_db.go:90]: map keys and values should be aligned by `gofmt`.

File: models/user_test.go (hunk 1 above)
Comment (potsbo) [reply models/user_test.go:39]: Instruction followed

File: store/dynamo_db.go (hunk 2 above)
Comment (potsbo) [reply store/dynamo_db.go:90]: Instruction followed

**Comment by potsbo** [comment]
Fixed them. It's ready.

File: models/user_test.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+func TestSlackMention(t *testing.T) {
+	user := SetUser()
+
+	expect := "<@SLACKID|slack_user>"
+	actual := user.SlackMention()
+
+	if actual != expect {
+		t.Fatalf("%v does not much to expected: %v", actual, expect)
```
Comment (dtan4) [inline models/user_test.go]: much -> match ❓

Comment (potsbo) [reply models/user_test.go]: Fixed, thank you

**Comment by potsbo** [comment]
@dtan4 It's ready

**Review by dtan4 — APPROVED** [review]

**Comment by dtan4** [comment]
LGTM 👍

**Pull request merged**